manifests/plugin/python.pp

```diff
@@ -1,45 +1,51 @@
 # See http://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_python
-define collectd::plugin::python (
-  $modulepath,
-  $module,
-  $script_source,
-  $ensure = present,
-  $config = {},
-  $order = '10',
+class collectd::plugin::python (
+  $modulepath = '/usr/lib/collectd/python',
+  $ensure     = present,
+  $modules    = {},
+  $order      = '10',
+  $interval   = undef,
 ) {
   include collectd::params
 
-  validate_hash($config)
+  validate_hash($modules)
 
   $conf_dir = $collectd::params::plugin_conf_dir
 
-  # This is deprecated file naming ensuring old style file removed, and should be removed in next major relese
-  file { "${name}.load-deprecated":
-    ensure => absent,
-    path   => "${conf_dir}/${name}.conf",
+  collectd::plugin {'python':
```

Comment:
Should we pass $globals ? 
http://collectd.org/documentation/manpages/collectd-python.5.shtml#configuration
https://github.com/pdxcat/puppet-module-collectd/blob/master/manifests/plugin.pp#L6